### PR TITLE
Lizenzangaben laut Projekt

### PR DESCRIPTION
**Pull request by hofmannedv**

### CODE REVIEW

File: docs/viz/overview.rst [hunk 1]
```diff
@@ -324,6 +324,15 @@ Altair seit mindestens einem Jahr nicht mehr weiterentwickelt wurden (Stand:
...
     "`ipyleaflet <https://github.com/jupyter-widgets/ipyleaflet>`_",".. image:: https://raster.shields.io/github/stars/jupyter-widgets/ipyleaflet",".. image:: https://raster.shields.io/github/contributors/jupyter-widgets/ipyleaflet",".. image:: https://raster.shields.io/github/commit-activity/y/jupyter-widgets/ipyleaflet",".. image:: https://raster.shields.io/github/license/jupyter-widgets/ipyleaflet"
 
+Anmerkungen zu Lizenzen laut Projektangaben:
+
+* Apache 2.0: bqplot, TensorFlow, Yellowbrick
+* BSD 2.0 : altair, Bokeh, d3po, geopandas, ggpy, glumpy, holoviews, mpld3, networkx, NumPy, Pandas, pythreejs, scikit-learn, SciPy, Seaborn, Statsmodels, SymPy, Theano, vega, VisPy 
+* LGPL 3.0 : cartopy, iris 
+* MIT: cufflinks, graphviz, ipyleaflet, ipyvolume, OpenCV, plotly, scikit-plot, vaex, vincent 
+* PSF: Matplotlib, Matplotlib Altair 
+* Andere Lizenz: datashader, toyplot, Vega-lite, yt
```
Comment (veit) [inline docs/viz/overview.rst]: datashader, toyplot, Vega-lite and yt appears to be licensed under a 3-clause BSD license.

File: docs/viz/overview.rst [hunk 2]
```diff
@@ -324,6 +324,15 @@ Altair seit mindestens einem Jahr nicht mehr weiterentwickelt wurden (Stand:
     "`toyplot <https://github.com/sandialabs/toyplot>`_",".. image:: https://raster.shields.io/github/stars/sandialabs/toyplot",".. image:: https://raster.shields.io/github/contributors/sandialabs/toyplot",".. image:: https://raster.shields.io/github/commit-activity/y/sandialabs/toyplot",".. image:: https://raster.shields.io/github/license/sandialabs/toyplot"
     "`ipyleaflet <https://github.com/jupyter-widgets/ipyleaflet>`_",".. image:: https://raster.shields.io/github/stars/jupyter-widgets/ipyleaflet",".. image:: https://raster.shields.io/github/contributors/jupyter-widgets/ipyleaflet",".. image:: https://raster.shields.io/github/commit-activity/y/jupyter-widgets/ipyleaflet",".. image:: https://raster.shields.io/github/license/jupyter-widgets/ipyleaflet"
 
+Anmerkungen zu Lizenzen laut Projektangaben:
+
+* Apache 2.0: bqplot, TensorFlow, Yellowbrick
+* BSD 2.0 : altair, Bokeh, d3po, geopandas, ggpy, glumpy, holoviews, mpld3, networkx, NumPy, Pandas, pythreejs, scikit-learn, SciPy, Seaborn, Statsmodels, SymPy, Theano, vega, VisPy 
+* LGPL 3.0 : cartopy, iris 
+* MIT: cufflinks, graphviz, ipyleaflet, ipyvolume, OpenCV, plotly, scikit-plot, vaex, vincent
```
Comment (veit) [inline docs/viz/overview.rst]: * Graphviz appears to be released under the Eclipse Public License 1.0 (EPL), see https://graphviz.gitlab.io/license/
* OpenCV under the 3-clause BSD License, see https://opencv.org/license/

File: docs/viz/overview.rst [hunk 3]
```diff
@@ -324,6 +324,15 @@ Altair seit mindestens einem Jahr nicht mehr weiterentwickelt wurden (Stand:
     "`toyplot <https://github.com/sandialabs/toyplot>`_",".. image:: https://raster.shields.io/github/stars/sandialabs/toyplot",".. image:: https://raster.shields.io/github/contributors/sandialabs/toyplot",".. image:: https://raster.shields.io/github/commit-activity/y/sandialabs/toyplot",".. image:: https://raster.shields.io/github/license/sandialabs/toyplot"
     "`ipyleaflet <https://github.com/jupyter-widgets/ipyleaflet>`_",".. image:: https://raster.shields.io/github/stars/jupyter-widgets/ipyleaflet",".. image:: https://raster.shields.io/github/contributors/jupyter-widgets/ipyleaflet",".. image:: https://raster.shields.io/github/commit-activity/y/jupyter-widgets/ipyleaflet",".. image:: https://raster.shields.io/github/license/jupyter-widgets/ipyleaflet"
 
+Anmerkungen zu Lizenzen laut Projektangaben:
+
+* Apache 2.0: bqplot, TensorFlow, Yellowbrick
+* BSD 2.0 : altair, Bokeh, d3po, geopandas, ggpy, glumpy, holoviews, mpld3, networkx, NumPy, Pandas, pythreejs, scikit-learn, SciPy, Seaborn, Statsmodels, SymPy, Theano, vega, VisPy
```
Comment (veit) [inline docs/viz/overview.rst]: * altair, Bokeh, d3po, glumpy, holoviews, mpld3, networkx, NumPy, Pandas , pythreejs, scikit-learn, SciPy, Seaborn, vega and VisPy appears to be released under the BSD 3-Clause License, see https://github.com/bokeh/bokeh/blob/master/LICENSE.txt, https://github.com/altair-viz/altair/blob/master/LICENSE, https://github.com/adrn/d3po/blob/master/LICENSE.txt, https://github.com/geopandas/geopandas/blob/master/LICENSE.txt, https://github.com/glumpy/glumpy/blob/master/LICENSE.txt, https://github.com/holoviz/holoviews/blob/master/LICENSE.txt, https://github.com/mpld3/mpld3/blob/master/LICENSE, https://github.com/networkx/networkx/blob/master/LICENSE.txt, https://github.com/numpy/numpy/blob/master/LICENSE.txt, https://github.com/pandas-dev/pandas/blob/master/LICENSE, https://github.com/jupyter-widgets/pythreejs/blob/master/LICENSE, https://github.com/scikit-learn/scikit-learn/blob/master/COPYING, https://github.com/scipy/scipy/blob/master/LICENSE.txt, https://github.com/mwaskom/seaborn/blob/master/LICENSE, https://github.com/vega/vega/blob/master/LICENSE and https://github.com/vispy/vispy/blob/master/LICENSE.txt
* Statsmodels, SymPy and Theano  are currently not discussed in the Jupyter tutorial

File: docs/viz/overview.rst [hunk 4]
```diff
@@ -324,6 +324,15 @@ Altair seit mindestens einem Jahr nicht mehr weiterentwickelt wurden (Stand:
...
     "`ipyleaflet <https://github.com/jupyter-widgets/ipyleaflet>`_",".. image:: https://raster.shields.io/github/stars/jupyter-widgets/ipyleaflet",".. image:: https://raster.shields.io/github/contributors/jupyter-widgets/ipyleaflet",".. image:: https://raster.shields.io/github/commit-activity/y/jupyter-widgets/ipyleaflet",".. image:: https://raster.shields.io/github/license/jupyter-widgets/ipyleaflet"
 
+Anmerkungen zu Lizenzen laut Projektangaben:
+
+* Apache 2.0: bqplot, TensorFlow, Yellowbrick
+* BSD 2.0 : altair, Bokeh, d3po, geopandas, ggpy, glumpy, holoviews, mpld3, networkx, NumPy, Pandas, pythreejs, scikit-learn, SciPy, Seaborn, Statsmodels, SymPy, Theano, vega, VisPy 
+* LGPL 3.0 : cartopy, iris 
+* MIT: cufflinks, graphviz, ipyleaflet, ipyvolume, OpenCV, plotly, scikit-plot, vaex, vincent 
+* PSF: Matplotlib, Matplotlib Altair
```
[the remark this answers is not in the document]
Comment (hofmannedv) [reply docs/viz/overview.rst]: well, I checked either the project sources on GitHub (LICENSE file), or the project website, or the according Wikipedia entry. It looks like that the information is not consistent. I suggest a re-check to be on the safe side.

[the remark this answers is not in the document]
Comment (hofmannedv) [reply docs/viz/overview.rst]: Also, the term "BSD license" is not specific enough. Maybe we divide it into BSD 2.0 and BSD 3.0 for clarity. What do you think?

[the remark this answers is not in the document]
Comment (veit) [reply docs/viz/overview.rst]: > Also, the term "BSD license" is not specific enough. Maybe we divide it into BSD 2.0 and BSD 3.0 for clarity. What do you think?

Ich würde, wie bei den Badges, unterscheiden zwischen _BSD-2-Clause_ und _BSD-3-Clause_.

[the remark this answers is not in the document]
Comment (hofmannedv) [reply docs/viz/overview.rst]: > mpl-altair appears to be licensed under a 2-clause BSD license, see https://github.com/matplotlib/mpl-altair/blob/master/LICENSE

OK, the we follow that

File: docs/viz/overview.rst (hunk 1 above)
Comment (hofmannedv) [reply docs/viz/overview.rst]: OK, I agree with you on that

**Comment by veit** [comment]
* The pyviz chapter has been moved to another repository: [PyViz-Tutorial](https://github.com/veit/pyviz-tutorial)
* The licenses should be recognised automatically

**Pull request closed without merging**